tests/TestCase.php

```diff
@@ -4,13 +4,15 @@
 
 namespace Tests;
 
+use Illuminate\Foundation\Testing\RefreshDatabase;
 use Illuminate\Foundation\Testing\TestCase as BaseTestCase;
 use Illuminate\Support\Facades\Bus;
 use Illuminate\Support\Facades\Event;
 
 abstract class TestCase extends BaseTestCase
 {
     use CreatesApplication;
+    use RefreshDatabase;
```

Comment:
I think I specifically didn't add `RefreshDatabase` to the base test case, because there are tests like the Mappers that don't need the database connection at all. The interaction with the database adds a overhead when wanting to run tests that don't use it.

Reply:
Good point! I will revert it again.